blocks/ai-chat/package.json

```diff
@@ -42,7 +42,7 @@
     "@types/react-dom": "18.0.9",
     "block-scripts": "0.2.3",
     "eslint": "8.33.0",
-    "mock-block-dock": "0.1.4",
+    "mock-block-dock": "0.1.3",
```

Comment:
Because it fixed [this](https://hashintel.slack.com/archives/C02K2ARC1BK/p1680105574802709) for me. Can undo it and see if it breaks my dev environment again? Have you encountered this on `0.1.4` by any chance?

Reply:
I confirm this happens in my environment as well with version `0.1.4`.